msf_updates/include/msf_updates/pose_sensor_handler/pose_measurement.h

```diff
@@ -295,22 +295,22 @@ struct PoseMeasurement : public PoseMeasurementBase {
       // Get rotation matrices.
       Eigen::Matrix<double, 3, 3> C_wv = state.Get<StateQwvIdx>()
 
-          .conjugate().toRotationMatrix();
-      Eigen::Matrix<double, 3, 3> C_q = state.Get<StateDefinition_T::q>()
-          .conjugate().toRotationMatrix();
+          .toRotationMatrix();
+      Eigen::Matrix<double, 3, 3> C_wi = state.Get<StateDefinition_T::q>()
+          .toRotationMatrix();
 
       // Construct residuals.
       // Position.
       r_old.block<3, 1>(0, 0) = z_p_
-          - (C_wv.transpose()
+          - (C_wv
               * (-state.Get<StatePwvIdx>()
                   + state.Get<StateDefinition_T::p>()
-                  + C_q.transpose() * state.Get<StatePicIdx>()))
+                  + C_wi * state.Get<StatePicIdx>()))
               * state.Get<StateLIdx>();
 
       // Attitude.
       Eigen::Quaternion<double> q_err;
-      q_err = (state.Get<StateQwvIdx>()
+      q_err = (state.Get<StateQwvIdx>().conjugate()
```

Comment:
isn't here （calculate the residual of position）?  p_vc = c_vw*(-p_wv+p_wi + c_wi*pic)*L = c_vw*(pw_vi + pw_ic)*L = c_vw*(pw_vc)*L = p_vc*L (pw_vi means p_vi expressed in the world frame)
is c_wv means the rotation matrix from vision frame to world frame ?
so why use the “C_wv” but not the "C_wv.transpose()" in line 305 ,what's more in function "CalculateH( ) " use" H.block<3, 3>(0, kIdxstartcorr_p) = C_wv.transpose( )" for calculation of the Jacobian